ewdistrict.py

```diff
@@ -575,50 +594,52 @@ def change_slimes(self, n = 0, source = None):
 	)
 
 	if len(users_in_district) > len(allies_in_district):
-		response = "You cannot capture a district while enemy gangsters are present."
+		response = "Holy shit, deal with your rival gangsters first! You can’t spray graffiti while they’re on the prowl!"
```

Comment:
this should only count pvp flagged enemies, since you can't get rid of unflagged ones